dissect/target/helpers/configutil.py

```diff
@@ -240,6 +240,35 @@ def parse_file(self, fh: TextIO) -> None:
         self.parsed_data = information_dict
 
 
+class Csvish(Default):
+    """Parses CSV-ish config files (does not confirm to CSV standard!)"""
+
+    def __init__(self, *args, **kwargs) -> None:
+        self.fields = kwargs.pop("fields")
```

Comment:
No fields are mandatory, but I can't override the generic constructor. I could add an exception, but it seems overkill, since the code will fail anyway.